src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -166,10 +173,11 @@ private Task ResetLockout(TUser user, CancellationToken token)
                 await UserManager.AccessFailedAsync(user, cancellationToken);
```

Comment:
you mean log intermediate method calls as well. Sure I think it is a good approach if the method call returns IdentityResult. I can do a pass